bindings/java/pom.xml

```diff
@@ -31,7 +31,7 @@
     </parent>
 
     <groupId>org.apache.opendal</groupId>
-    <artifactId>opendal-java</artifactId>
+    <artifactId>opendal</artifactId>
     <version>0.46.4</version>
```

Comment:
Next version should be `0.47.0` then. I tend to update the version alongside with the PR .. But it's up to our current flavor.

Reply:
> Next version should be `0.47.0` then. I tend to update the version alongside with the PR .. But it's up to our current flavor.

We will bump this while release.